playground/package.json

```diff
@@ -20,7 +20,7 @@
     "@commercetools-frontend/mc-http-server": "3.0.0",
     "@commercetools-frontend/permissions": "3.0.0",
     "@commercetools-frontend/sdk": "3.0.0",
-    "@commercetools-frontend/ui-kit": "2.0.0-rc.11",
+    "@commercetools-frontend/ui-kit": "4.0.2",
```

Comment:
@montezume This is quite the jump here. Did you try the playground out?

Reply:
yessir. It doesn't even use ui-kit. Maybe I should just remove it?

Reply:
yeah I tried it out. We are only using ~4 components, and none of those had changes

  LoadingSpinner, Table, Spacings, Constraints, Text